fsck.c

```diff
@@ -1323,6 +1337,62 @@ int fsck_finish(struct fsck_options *options)
 	return ret;
 }
 
+__attribute__((format (printf, 4, 5)))
```

Comment:
The `__attribute__` part should only exist in the declaration, not in the definition of this function.